miscellaneous-commands/userconfig.md

```diff
@@ -22,8 +22,9 @@ For more information on editing permission requirements for specific users/roles
 /userconfig settings [reaction-role-notifications] [vote-reminders]
 ```
 
-* `reaction-role-notifications`: <TODO: describe>
-* `vote-reminders`: <TODO: describe>
+* `reaction-role-notifications`: This subcommand lets you enable or disable notifications when you add or remove [a reaction role](/start-up/setting-up-reaction-roles.md) from yourself.
+* `vote-reminders`: This subcommand will remind you daily to [vote for MoonlightBot.](/MoonlightBot-docs/support/upvote-moonlightbot.md)
```

Comment:
default is true

also the full stops

reminders are triggered by the act of voting itself, not using this command